### ISSUE TITLE
Map Broken In Spanish

### ISSUE THREAD

**Issue by reustle** [reustle/covid19japan#300]
Seems that the language switching might not be handled properly in the mapbox related code.

<img width="674" alt="image" src="https://user-images.githubusercontent.com/304560/80938600-b24b7080-8e14-11ea-9e39-a2e64833c4b0.png">

**Comment by liquidx** [comment]
Does it load if you zoom in? 

I seem to see the same issue on Japanese too. I wonder if it's a Mapbox bug?

**Comment by reustle** [comment]
You're right, broken on Japanese too. This is a new bug then, as previously it did switch fairly quickly without any interaction with the map.

You can see a discussion about language switching here https://github.com/mapbox/mapbox-gl-js-docs/issues/41

It's where I posted my (hacky) solution

**Comment by liquidx** [comment]
There's a suggestion in there to try https://github.com/mapbox/mapbox-gl-language

Did we ever try that? 

Odd thing is that when I first observed this bug, changing the zoom level will show the labels. But now, changing the zoom levels still don't show the labels.

**Comment by liquidx** [comment]
I changed to start from zoom level 3, and it seems like zoom level 4 doesn't work, but zoom level 5 shows the labels again. I'm mystified as to what's happening.

![map 2020-05-04 23_36_51](https://user-images.githubusercontent.com/72062/80977803-34ad5200-8e60-11ea-93d4-0ea7d1ee27c4.gif)
